docs/download/download.md

```diff
@@ -0,0 +1,24 @@
+# Download
+
+# Verification
+ you can follow these [procedures](https://www.apache.org/dyn/closer.cgi#verify) and the [KEYS](https://dist.apache.org/repos/dist/dev/incubator/weex/KEYS) file to verify the download files
+
+
+# Apache Weex
+> GitHub: [https://github.com/apache/incubator-weex](https://github.com/apache/incubator-weex)    
+> Release Notes: [https://github.com/apache/incubator-weex](https://github.com/apache/incubator-weex/releases)
+
+## 0.20.0
+- [source](https://dist.apache.org/repos/dist/dev/incubator/weex/0.20.0/RC1/apache-weex-incubating-0.20.0-RC1-src.tar.gz) | [asc](https://dist.apache.org/repos/dist/dev/incubator/weex/0.20.0/RC1/apache-weex-incubating-0.20.0-RC1-src.tar.gz.asc) | [sha512](https://dist.apache.org/repos/dist/dev/incubator/weex/0.20.0/RC1/apache-weex-incubating-0.20.0-RC1-src.tar.gz.sha512)

```

Comment:
According to [release-download-pages](http://www.apache.org/dev/release-download-pages), using mirror address like http://www.apache.org/dyn/closer.cgi?filename=incubator/weex/0.20.0/apache-weex-incubating-0.20.0-src.tar.gz&action=download would be more suitable.

Reply:
done